.circleci/deploy-config.yml

```diff
@@ -99,17 +102,19 @@ jobs:
           name: Pre-test checks
           command: |
             . .venv/bin/activate
-            black --check .
             python -m pip check
             python manage.py check
             python manage.py makemigrations --check
+            ruff format . --check
+            ruff check .
+            git ls-files '*.html' | xargs djhtml --check
```

Comment:
Exact same comment as https://github.com/DemocracyClub/EveryElection/pull/2210#discussion_r1718178871 for the same reasons